applications/asset_tracker_v2/src/modules/modem_module.c

```diff
@@ -855,6 +854,13 @@ static int setup(void)
 		return err;
 	}
 
+	/* Setup a callback for the default PDP context. */
+	err = pdn_default_callback_set(pdn_event_handler);
```

Comment:
uhm, I am thinking we might need to extend this to allow for multiple callbacks.
Right now this is also set by `libcarrier` in `lwm2m_os_pdn_default_callback_set()`

Reply:
Will make the appropriate changes once https://github.com/nrfconnect/sdk-nrf/pull/8119 is merged.